lib/dslash_domain_wall_4d.cu

```diff
@@ -215,6 +215,28 @@ namespace quda {
       }
       return flops;
     }
+
+    long long bytes() const {
+      bool isHalf = in->Precision() == sizeof(short) ? true : false;
+      int spinor_bytes = 2 * in->Ncolor() * in->Nspin() * in->Precision() + (isHalf ? sizeof(float) : 0);
+      long long Ls = in->X(4);
+      long long bytes;
+
+      switch(DS_type){
```

Comment:
While this is not introduced here and does not stop me from merging this pull request: is an enum a better choice for DS_type? In the long haul we might go for strongly typed C++11 enum but that is some time to go until we can drop support for CUDA versions without C++11 support.